sdk/anomalydetector/ai-anomaly-detector/README.md

```diff
@@ -0,0 +1,124 @@
+
+```PowerShell
+az cognitiveservices account create --kind AnomalyDetector --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+### Install the `@azure/ai-anomaly-detector` package
+
+Install the Azure Anomaly Detector client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/ai-anomaly-detector
+```
+
+### Create and authenticate a `AnomalyDetectorClient`
+
+To create a client object to access the Anomaly Detector API, you will need the `endpoint` of your Anomaly Detector resource and a `credential`. The Anomaly Detector client can use either Azure Active Directory credentials or an API key credential to authenticate.
+
+You can find the endpoint for your Anomaly Detector resource either in the [Azure Portal][azure_portal] or by using the [Azure CLI][azure_cli] snippet below:
+
+```bash
+az cognitiveservices account show --name <your-resource-name> --resource-group <your-resource-group-name> --query "endpoint"
+```
+
+#### Using an API Key
+
+Use the [Azure Portal][azure_portal] to browse to your Anomaly Detector resource and retrieve an API key, or use the [Azure CLI][azure_cli] snippet below:
+
+**Note:** Sometimes the API key is referred to as a "subscription key" or "subscription API key."
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+Once you have an API key and endpoint, you can use the `AzureKeyCredential` class to authenticate the client as follows:
+
+```js
+const { AnomalyDetectorClient, AzureKeyCredential } = require("@azure/ai-anomaly-detector");
+
+const client = new AnomalyDetectorClient("<endpoint>", new AzureKeyCredential("<API key>"));
+```
+
+#### Using an Azure Active Directory Credential
+
+Client API key authentication is used in most of the examples, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][defaultazurecredential] provider shown below,
+or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
+
+```bash
+npm install @azure/identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and grant access to Anomaly Detector by assigning the `"Cognitive Services User"` role to your service principal (note: other roles such as `"Owner"` will not grant the necessary permissions, only `"Cognitive Services User"` will suffice to run the examples and the sample code).
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+```js
+const { AnomalyDetectorClient } = require("@azure/ai-anomaly-detector");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+const client = new AnomalyDetectorClient("<endpoint>", new DefaultAzureCredential());
+```
+
+## Examples
+
+Coming soon
```

Comment:
sometimes it's useful to link to a work item for this work in order to make sure we don't forget to come back and do it ;)